<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="determiantLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Determinant =</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newMatrixButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt;
background-color: &quot;crimson&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>New matrix</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Here comes solution</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
